recipes/darts/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pt.darts" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name.replace("pt.", "") }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/khanrc/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b0c4689cdabc5fdec0ba3c829057aa2cb94f9a97036cf5ae545de6247d301e9c
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
It looks like this is pure Python? You should be able to make this `noarch: python`, remove the `skip`, and we can ignore the build failure due to the lack of PyTorch on Windows.

Reply:
noarch added. skip removed.